coldfront/core/user/templates/request_hub/request_hub.html

```diff
@@ -0,0 +1,123 @@
+{% extends "common/base.html" %}
+{% load common_tags %}
+{% block content %}
+
+  <h1>Request Hub</h1>
+
+  <div class="table-responsive">
+    <table class="table table-sm">
+      <tbody>
+      <tr>
+        <td>
+          {% if show_all %}
+              Below are all of the requests in the myBRC Portal. Click on a request
+              section to view requests of that type. To perform actions on a specific request,
+              click the button to go to the request's main page and perform the actions there.
+              Click <a href="{% url 'request-hub' %}">here</a> to view only your requests.
+          {% else %}
+              Below are all of your requests in the myBRC Portal. Click on a request
+              section to view your requests of that type.
+
+              {% if admin_staff %}
+                Click <a href="{% url 'request-hub-admin' %}">here</a> to view all requests in the myBRC Portal.
+              {% endif %}
+          {% endif %}
+        </td>
+      </tr>
+      <tr><td></td></tr>
+      </tbody>
+    </table>
+  </div>
+  <div class="card mb-3 bg-light">
+    <div class="card-header">
+      <div class="float-right">
+        <button id="collapse_all_button" type="button" class="btn btn-primary">
+          Collapse All
+          <i class="fas fa-minus" aria-hidden="true"></i>
+        </button>
+        <button id="expand_all_button" type="button" class="btn btn-primary">
+          Expand All
+          <i class="fas fa-plus" aria-hidden="true"></i>
+        </button>
+      </div>
+    </div>
+  </div>
+
+  {% with request_obj=cluster_account_request_obj %}
+    {% include 'request_hub/request_section.html' %}
+  {% endwith %}
+
+  {% with request_obj=project_join_request_obj %}
+    {% include 'request_hub/request_section.html' %}
+  {% endwith %}
+
+  {% with request_obj=project_removal_request_obj %}
+    {% include 'request_hub/request_section.html' %}
+  {% endwith %}
+
+  {% with request_obj=project_renewal_request_obj %}
+    {% include 'request_hub/request_section.html' %}
+  {% endwith %}
+
+  {% with request_obj=savio_project_request_obj %}
+    {% include 'request_hub/request_section.html' %}
+  {% endwith %}
+
+  {% with request_obj=su_purchase_request_obj %}
+    {% include 'request_hub/request_section.html' %}
+  {% endwith %}
+
+  {% with request_obj=vector_project_request_obj %}
+    {% include 'request_hub/request_section.html' %}
+  {% endwith %}
+
+  <script>
+    {# set variables to be used for scroll position #}
```

Comment:
Cool, I imagine having the same sections open when returning to this later could be useful.

Reply:
It was more because when we click on the next page, there's technically a reload so the scroll position was reset. It was super annoying if you were anywhere other than the top of the page trying to click through pages